README.md

```diff
@@ -9,12 +9,79 @@
+
+```html
+<html class="spectrum spectrum--medium spectrum--light">
+```
+
+Use the `index.css` files in your project to include component styles and all of the global context variable definitions (Spectrum and Express contexts, background theme/colorstop, platform scaling, etc.) for the component (if you don't need all of the global context definitions, peek at the docs for [including assets](https://github.com/adobe/spectrum-css?tab=readme-ov-file#including-assets)). Use this file by including the stylesheet (plus the stylesheets for the dependencies) in the `<head>` tag:
+
+```html
+<head>
+  <!-- Include global tokens first -->
+  <link rel="stylesheet" href="node_modules/@spectrum-css/tokens/dist/index.css"/>
+
+  <!-- Include index.css for all components you're using -->
+  <link rel="stylesheet" href="node_modules/@spectrum-css/page/dist/index.css"/>
+  <link rel="stylesheet" href="node_modules/@spectrum-css/button/dist/index.css"/>
+</head>
+```
+
+Inside the <body> tag, add the markup for your component (Spectrum button in our example). The example also includes the CSS class names spectrum-Button--fill spectrum-Button--accent, to use the accent variant:
+
+```html
+<button class="spectrum-Button spectrum-Button--fill spectrum-Button--accent spectrum-Button--sizeM">
+  <span class="spectrum-Button-label">Button</span>
+</button>
+```
+
+To put it all together, your final html file will look like this:
+
+```html
+<html class="spectrum spectrum--light spectrum--medium">
+  <head>
+    <link rel="stylesheet" href="node_modules/@spectrum-css/tokens/dist/index.css"/>
+    <link rel="stylesheet" href="node_modules/@spectrum-css/page/dist/index-vars.css">
+    <link rel="stylesheet" href="node_modules/@spectrum-css/button/dist/index-vars.css">
+  </head>
+  <body>
+    <button class="spectrum-Button spectrum-Button--fill spectrum-Button--accent spectrum-Button--sizeM">
+      <span class="spectrum-Button-label">Button</span>
+    </button>
+  </body>
+</html>
+```
+
+### CDN
+Another way to include Spectrum CSS components in your project is to use a CDN, [unpkg](https://unpkg.com/) for example, to link to the components you want, plus their dependencies, in the `<head>` tag of your HTML. Here's an example using the button component:
+
+```html
+<head>
+  <!-- Include global tokens first -->
+  <link rel="stylesheet" href="https://unpkg.com/browse/@spectrum-css/tokens/"/>
+
+  <!-- Include index.css for all components you're using -->
+  <link rel="stylesheet" href="https://unpkg.com/browse/@spectrum-css/page/"/>
+  <link rel="stylesheet" href="https://unpkg.com/browse/@spectrum-css/button/"/>
+</head>
+```
```

Comment:
One of our [users requested](https://adobedesign.slack.com/archives/C5N154FEY/p1710946376240839?thread_ts=1710340701.269509&cid=C5N154FEY) we show how to use a CDN to include Spectrum CSS in a project. Is this how we would do that?

Reply:
I suggested changes for the URLs, but yes, this is mostly correct. We need to point _directly_ to a specific CSS asset in the `dist`, and here I'm suggesting that it's the `index.css` that folks would want to use.

Also, unpkg should just resolve the request to the latest version that's been published to npm, so no need to include the actual version number (unless someone wanted to use a specific version number)